parm/soca/obsprep/obsprep_config.yaml

```diff
@@ -149,3 +149,21 @@ observations:
     binning:
       stride: 15
       min number of obs: 10
+- obs space:
+    name: insitu_profile_bathy
+    provider: GTS
+    dmpdir subdir: atmos
+    output file: gdas.t00z.bathy_profile.tesac.nc
```

Comment:
Is it hard coded because it's coming from a "daily tank" @ShastriPaturi ?
That's not going to work for the `gfs` cycle.

Reply:
Hard coded just like the others and will not be used. The 6-hrly dumps have names based on the `RUN`.

Reply:
you need to drop `gdas.t00z`. That can't be hard coded, after being processed, the files are moved to the "com obs" and renamed following the convention we're supposed to be using.